babel.config.js

```diff
@@ -12,10 +12,10 @@ module.exports = {
       "@babel/preset-env",
       {
         useBuiltIns: "usage",
-        corejs: { version: "3.9", proposals: true },
+        corejs: { version: "3.26", proposals: true },
         targets: {
-          node: "12",
-          browsers: ["> 0.2% and not dead", "firefox >= 44"],
+          node: "16",
+          browsers: ["> 0.2%", "last 2 versions", "not dead", "Firefox ESR"],
```

Comment:
Our contributor CatQuest uses FireFox 44, so I'd like to see that version kept in the list.

Reply:
Alright, I'm on it